frontend/src/plugins/impl/NumberPlugin.tsx

```diff
@@ -44,30 +47,16 @@ interface NumberComponentProps extends Data {
 
 const NumberComponent = (props: NumberComponentProps): JSX.Element => {
   const inputRef = useRef<HTMLInputElement>(null);
-  const oninput = () => {
-    if (inputRef.current !== null) {
-      props.setValue(inputRef.current.valueAsNumber);
-    }
-  };
-
-  useEffect(() => {
-    // scrolling inside the input causes the value to change, which
-    // is very surprising, so we disable it.
-    if (inputRef.current !== null) {
-      inputRef.current.addEventListener(
-        "wheel",
-        (e) => {
-          if (e.target === inputRef.current) {
-            e.preventDefault();
-          }
-        },
-        { passive: false }
-      );
```

Comment:
Can you disable/ignore scrolling again? This was based on feedback

Reply:
i moved it to:
```
        onWheel={(e) => {
          e.preventDefault();
        }}
```

were you hitting this issue still with that change? i tried to repro and couldnt

Reply:
Oh weird. I thought I was. But now I can't repro ... my bad.